### PR TITLE
Added missing translations (Italian)

### PR DESCRIPTION
**Pull request by silvio-didonna**

### CODE REVIEW

File: Marlin/language_it.h [hunk 1]
```diff
@@ -174,5 +176,57 @@
...
+#define MSG_INFO_STATS_MENU                 "Statistiche"
+#define MSG_INFO_BOARD_MENU                 "Info. scheda"
+#define MSG_INFO_THERMISTOR_MENU            "Termistori"
+#define MSG_INFO_EXTRUDERS                  "Estrusori"
+#define MSG_INFO_BAUDRATE                   "Baud"
+#define MSG_INFO_PROTOCOL                   "Protocollo"
+#if LCD_WIDTH > 19
+  #define MSG_INFO_PRINT_COUNT              "Contat. stampa"
+  #define MSG_INFO_FINISHED_PRINTS          "Finiti      "
+  #define MSG_INFO_PRINT_TIME               "Tempo totale"
```
Comment (thinkyhead) [inline Marlin/language_it.h]: These 3 strings are meant to align. Please add a trailing space to each of the two the shorter strings.

Comment (silvio-didonna) [inline Marlin/language_it.h]: @thinkyhead Please check if it is ok now.

**Comment by silvio-didonna** [comment]
Why have you closed this pull request?
I made the modifications suggested.

There is some errors in my PR?

**Comment by jbrazio** [comment]
@tnw513 look at #4329, it was a rebase of this one so it could be merged.

**Comment by silvio-didonna** [comment]
I see, but why an other PR? I've made a mistake?
Probably I miss something in how git rebase works. In this case why "rebase" and not "merge"?

**Comment by jbrazio** [comment]
"Rebase" must be done in the command line git and basically means that you want to update your working tree to the latest RCBugfix (the base) and then re-apply your changes on top of it.

If you look at the #4329 the change credit is still assign to your username, so it's exactly the same thing as merging this one but some times we cannot automatically merge changes using the Github web interface due to conflicts. We could ask the person to rebase the PR or in this case we do it ourselves.

**Comment by silvio-didonna** [comment]
Ok, perfect.
I'll rebase it next time, so you do not have to do more work than needed.

No problem for credit, it's only a translation :)

**Comment by jbrazio** [comment]
Just to make it all clear. :-D

**Pull request closed without merging**